randovania/games/prime2/logic_database/Temple Grounds.txt

```diff
@@ -1441,11 +1441,9 @@ Extra - in_dark_aether: False
           Morph Ball and After Ing Windchamber Yellow Seeker Lock and After Ing Windchamber Blue Seeker Lock
           Any of the following:
               Grapple Beam or Use Screw Attack (Space Jump)
-              All of the following:
-                  Space Jump Boots
-                  Any of the following:
-                      Scan Visor and Combat/Scan Dash (Advanced)
-                      Terminal Fall Abuse (Beginner) and Normal Damage ≥ 10
+              Scan Visor and Space Jump Boots and Combat/Scan Dash (Advanced)
+              # Jump to center platform, then towards platform with the cannon https://youtu.be/_Cs-JtpIuNE
+              Space Jump Boots and Terminal Fall Abuse (Beginner) and Normal Damage ≥ 10
```

Comment:
It is good, but an argument can be made that it should require Standable Terrain. Would like others' thoughts.

Reply:
Standable Terrain for whatm

Reply:
standing on the center platform https://discord.com/channels/458432177223368704/499458280108589061/1260277202818302063
my pov was that it's an intentional platform and doesn't need the trick but I won't fight back if folks disagree